### ISSUE TITLE
FootnoteZone?

### ISSUE THREAD

**Issue by dstoekl** [SegmOnto/Guidelines#47]
Is there a reason for not having a FootnoteZone and a CriticalApparatusZone for scanned books?

**Comment by Jean-Baptiste-Camps** [comment]
Question already asked repeatedly by @PonteIneptique, and discussed in #19 . Perhaps the choice we made is debatable, but since footnote and apparatus are semantic types, we relegated them to a subtype of `MarginText`.

**Comment by PonteIneptique** [comment]
This means one should use `MarginTextZone:variants` or `MarginTextZone:note` (I assume one could specify footnote).

**Comment by gabays** [comment]
Yes. Remember subtypes are only proposed, but you can always creates your own, if needed. `footnotes`, `endnotes`, `criticalApparatus` if it is impossible to disentangle things…
